android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIDrawerLayout.java

```diff
@@ -530,6 +532,23 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 		}
 	}
 
+	private void setDrawerLockMode(Object param){
+		// reset first
+		layout.setDrawerLockMode(DrawerLayout.LOCK_MODE_UNLOCKED);
+		if (param instanceof HashMap) {
+			HashMap<String, Object> drawerParams = (HashMap<String, Object>) param;
+			if (TiConvert.toInt(drawerParams.get(TiC.PROPERTY_GRAVITY)) == 1) {
```

Comment:
Use the constant `GRAVITY_BOTH` instead of `1`

NOTE: you could also use a `KrollDict` and use `getInt(TiC.PROPERTY_GRAVITY)` instead of needing to use `TiConvert`